### PR TITLE
update add-on and fix minor style issues

### LINKED ISSUE #595: Tidy up add-on for new UI

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #595

### CODE REVIEW

File: testpilot/frontend/static-src/app/templates/landing-page.js [hunk 1]
```diff
@@ -115,17 +116,17 @@ export default `
...
             <h2 data-l10n-id="landingInstallMessage" class="emphasis">Test Pilot installed!</h2>
-            <a href="/" data-l10n-id="landingInstalledButton" class="button large default quick-pop no-display">Let's Go</a>
+            <a href="/" data-l10n-id="landingInstalledButton" class="button large primary quick-pop no-display">Let's Go</a>
           </div>
           <div data-hook="default-message">
             <div class="copter-wrapper fly-down">
               <div class="copter"></div>
             </div>
-            <h2 class="banner">
+            <h2git  class="banner">
```
Comment (pdehaan) [inline testpilot/frontend/static-src/app/templates/landing-page.js]: @johngruen What dis?

Comment (johngruen) [inline testpilot/frontend/static-src/app/templates/landing-page.js]: that's where i had focus on the wrong thing when i typed git

**Comment by pdehaan** [comment]
Semi-related, but do we want to change these assets as well?

https://github.com/mozilla/testpilot/blob/addon-and-style-tweaks/addon/data/icon-16.png
https://github.com/mozilla/testpilot/blob/addon-and-style-tweaks/addon/data/icon-32.png
https://github.com/mozilla/testpilot/blob/addon-and-style-tweaks/addon/data/icon-64.png

**Comment by johngruen** [comment]
@pdehaan yeah, bryan's got one in progress

File: testpilot/frontend/static-src/styles/modules/_modals.scss [hunk 2]
```diff
@@ -37,6 +37,13 @@
   position: relative;
   width: $grid-unit * 21;
 
+  :not(.content-wrapper) & {
+    @include respond-to('small') {
+      width: 92%;
+      margin: 0 4%;
```
Comment (pdehaan) [inline testpilot/frontend/static-src/styles/modules/_modals.scss:42]: Final nit, then I'll shut up and go back to eating donuts:

``` sh
npm run lint

> testpilot@1.0.0 lint /home/ubuntu/testpilot
> gulp lint sass-lint

[20:46:04] Using gulpfile ~/testpilot/gulpfile.js
[20:46:04] Starting 'lint'...
[20:46:04] Starting 'sass-lint'...

testpilot/frontend/static-src/styles/modules/_modals.scss
  42:7  warning  Expected `margin`, found `width`  property-sort-order
  43:7  warning  Expected `width`, found `margin`  property-sort-order

✖ 2 problems (0 errors, 2 warnings)
```

**Pull request merged**